src/hooks/useGetHasBottomSpace.ts

```diff
@@ -0,0 +1,16 @@
+import { useGetElemPositions } from './useGetElemPositions'
+import { useGetElemSizes } from './useGetElemSizes'
+
+export const useGetHasBottomSpace = ({
+  dropdownContainer,
```

Comment:
I think this is a generic property, so you can rename it, for example elementContainer or just element